Protocol.md

```diff
@@ -81,6 +82,11 @@ the funding transaction and CETs.
    * [`u64`:`feerate_per_vb`]
    * [`u32`:`cet_locktime`]
    * [`u32`:`refund_locktime`]
+   * [`offer_tlvs`: `tlvs`]
+
+[//]: # (TODO: `protocol_version` should be advertised by DLC nodes in the future)
+The `protocol_version` value denotes the version of the protocol that the offering node is proposing.
+If the receiving node does not support the requested version, it should ignore the message.
```

Comment:
Maybe we should wait for the `channel_type` discussion to settle on the LN-side and piggy-back there, defining new DLC features as-yet-another-channel ? 

See https://github.com/lightningnetwork/lightning-rfc/pull/880 and this week LN meeting : https://gnusha.org/lightning-dev/2021-07-05.log

Reply:
Can you clarify what you would like to do here?